streams/src/test/java/org/apache/kafka/streams/state/internals/ThreadCacheTest.java

```diff
@@ -511,6 +511,16 @@ public void shouldReturnNullIfKeyIsNull() {
         assertNull(threadCache.get(namespace, null));
     }
 
+    @Test
+    public void shouldTestSizeInBytes() {
+        final ThreadCache cache = new ThreadCache(logContext, 100000, new MockStreamsMetrics(new Metrics()));
+        cache.put(namespace1, Bytes.wrap(new byte[]{1}), cleanEntry(new byte[] {1}));
+        cache.put(namespace2, Bytes.wrap(new byte[]{1}), cleanEntry(new byte[] {1}));
+        assertEquals(cache.sizeBytes() + cache.size(), 2 * memoryCacheEntrySize(new byte[1], new byte[1], ""));
```

Comment:
Why are we not just checking the `sizeInBytes` ? Cache.size returns the number of items in the cache and is unrelated

Reply:
@dguy sure will take care.